srcpkgs/gajim/template

```diff
@@ -18,14 +18,14 @@ license="GPL-3.0-only"
 homepage="https://www.gajim.org"
 changelog="https://dev.gajim.org/gajim/gajim/-/raw/master/ChangeLog"
 distfiles="https://gajim.org/downloads/${version%.*}/gajim-${version}.tar.gz"
-checksum=4f15b2503a01bec70a6b6a203eb165ccb0b6abee513323270223a941da453f83
+checksum=649cf4ac65d096097ced9d5189521ae86554ba390370a1ffe9fdd175947f956c
 
 post_build() {
-	./pep517build/build_metadata.py -o dist/metadata
+	./make.py build --dist=unix
```

Comment:
Consider adding a prefix to install icons, desktop, man pages whatever else upstream may want to in the future
```suggestion
	./make.py build --dist=unix --prefix "${DESTDIR}/usr"
```